src/config/mozambique/translation.json

```diff
@@ -0,0 +1,96 @@
+{
+    "date_locale": "eu",
```

Comment:
I don't have a really good idea on how to do it yet, but I think we should try and organize this file in sections.

// Application

// Layers and Legends

.../...

To make it a bit easier to navigate. What do you think?

Reply:
good point. copying your suggestion from our conversation just now, comments could be added using something like:
“_application”: “section_comment”

will add this in the next iteration